core/src/avm2/globals/flash/display/bitmap.rs

```diff
@@ -87,8 +87,14 @@ pub fn init<'gc>(
             let bitmap_data = bitmap_data
                 .unwrap_or_else(|| BitmapDataWrapper::dummy(activation.context.gc_context));
 
-            let bitmap =
-                Bitmap::new_with_bitmap_data(&mut activation.context, 0, bitmap_data, smoothing);
+            let bitmap = Bitmap::new_with_bitmap_data(
+                activation.context.gc_context,
+                0,
+                bitmap_data,
+                smoothing,
+                activation.context.swf,
```

Comment:
(Nit: Would it be cleaner or less clean to keep both of the "context" parameters at the start? Like, instead of:

```rust
Bitmap::new_with_bitmap_data(activation.context.gc_context, 0, bitmap_data, smoothing, activation.context.swf);
```

it would be:


```rust
Bitmap::new_with_bitmap_data(activation.context.gc_context, activation.context.swf, 0, bitmap_data, smoothing);
```

Reply:
(Never mind, I think I misunderstood the code).